src/shaders/stdosl.h

```diff
@@ -530,7 +531,7 @@ int iscameraray () { return raytype("camera"); }
 int isdiffuseray () { return raytype("diffuse"); }
 int isglossyray () { return raytype("glossy"); }
 int isshadowray () { return raytype("shadow"); }
-int getmatrix (string fromspace, string tospace, output matrix M) BUILTIN;
+int getmatrix (string fromspace, string tospace, output matrix M) BUILTIN_NONSTANDARD_RW;
```

Comment:
That's a good point, it should be able to figure it out on its own. I'll look into that.